go/libraries/doltcore/sqle/dfunctions/verify_constraints.go

```diff
@@ -33,6 +33,7 @@ const (
 )
 
 // ConstraintsVerifyFunc represents the sql functions "verify_constraints" and "verify_constraints_all".
+// Deprecated: please use the version in the dprocedures package
```

Comment:
Same, no need to convert to stored procedure

Reply:
This writes to the constraint violations table, so it's not necessarily read-only.